scripts/benchmark-size/scope.json

```diff
@@ -0,0 +1,51 @@
+[
+  { "package": "@aws-sdk/credential-providers" },
+  { "package": "@aws-sdk/credential-provider-cognito-identity" },
+  { "package": "@aws-sdk/credential-provider-env", "skipBundlerTests": true },
+  { "package": "@aws-sdk/credential-provider-imds", "skipBundlerTests": true },
+  { "package": "@aws-sdk/credential-provider-ini", "skipBundlerTests": true },
+  { "package": "@aws-sdk/credential-provider-node", "skipBundlerTests": true },
+  { "package": "@aws-sdk/credential-provider-process", "skipBundlerTests": true },
+  { "package": "@aws-sdk/credential-provider-sso", "skipBundlerTests": true },
+  { "package": "@aws-sdk/credential-provider-web-identity" },
+  { "package": "@aws-sdk/abort-controller" },
+  { "package": "@aws-sdk/fetch-http-handler" },
+  { "package": "@aws-sdk/node-http-handler", "skipBundlerTests": true },
+  { "package": "@aws-sdk/polly-request-presigner" },
+  { "package": "@aws-sdk/s3-presigned-post" },
+  { "package": "@aws-sdk/s3-request-presigner" },
+  { "package": "@aws-sdk/signature-v4" },
+  { "package": "@aws-sdk/signature-v4-crt", "skipBundlerTests": true },
+  { "package": "@aws-sdk/smithy-client" },
+  { "package": "@aws-sdk/types" },
+  {
+    "package": "@aws-sdk/lib-dynamodb",
+    "dependencies": [
```

Comment:
dependencies can be removed, and instead, read the `peerDependencies` from the package.json